src/constants.py

```diff
@@ -16,13 +16,18 @@
 all                                         : prints all saved contacts
 add-birthday [DD.MM.YYYY]                   : add birthday to the contact
 show-birthday [username]                    : show birthday of the contact
-birthdays [range]                       : show upcoming birthdays in the next 7 days or, if you provide range, in the next range days
+birthdays [range]                           : show upcoming birthdays in the next 7 days or, if you provide range, in the next range days
+add-note [name] [text]                      : saves new note
```

Comment:
Do we need to have `name` paramete? As far as I understand the consept of this personal assistance's app there are two main entities AddressBook and NoteBook. You have succesfully implemented them. If the first one realy needs  to have the `name` parameter because we suppose to have many friend contacts, the last one (NoteBook) is dedicated to be used just for the single user help. To put it another way, all the notes are supposed to be created by me (the owner of the personal assistant).

I recollect that the structure of the data in our pre-command work meeting was similar to that:
{
  "address-book: {...},
  "note-book: {
      "1": {
              "note": "Do home work",
              ....
              },
         "2": {
              "note": "Do shoping",
              ....
        }
}

What do you thing it's reasonable not to have `name` arg in turns of the note creation but have the possible command like this 'add-note [text]' or 'add-note [title] [body]?

Reply:
argument "name" means the name of the Note, or "title", and has nothing to do with the contact name. This argument naming was chosen just to be consistent

Reply:
Ok, thank you. Now I see. 
Optional: it'd be clear for me if we use parameter title and body instead of name and text because the word 'name' leads me to the wrong understandiig.